### DISCUSSION TITLE
Can I turn off the grid lines?

### DISCUSSION THREAD

**Discussion by codinalot** [mesa/mesa#1567] [Q&A]
Hi all - question in the title. I'd like to be able to turn off the grid lines in the canvas grid so that the simulation can appear "continuous" in nature. Thanks in advance!

**Comment by ameligrana** [comment]
As a simple solution you can delete the line

`canvasDraw.drawGridLines("#eee");`

in `CanvasModule.js` and then reload the page after emptying the cache, hope this helps!
